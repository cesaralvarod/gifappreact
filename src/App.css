.container {
  padding: 3rem;
  margin: 0 auto;
  max-width: 120rem;
}
