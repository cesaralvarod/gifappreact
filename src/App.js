import React, { useState } from "react";

// Styles

import "./App.css";

// Component

import SearchInput from "./components/SearchInput";
import GifGrid from "./components/GifGrid";

const App = () => {
  const [query, setQuery] = useState("");

  const handleQuery = (value) => {
    setQuery(value);
  };

  return (
    <div className="container">
      <h1>Gif App React</h1>
      <SearchInput handleQuery={handleQuery} />
      <div className="content">
        <GifGrid query={query} />
      </div>
    </div>
  );
};

export default App;
