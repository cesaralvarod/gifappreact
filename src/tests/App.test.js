/*
 * Enzyme, Enzyme to json
 * */

import React from "react";
import { shallow } from "enzyme";
import App from "../App";

describe("Testing in App.js", () => {
  test("Component <App/> should show correctly", () => {
    const wrapper = shallow(<App />);

    expect(wrapper).toMatchSnapshot();
  });
});
