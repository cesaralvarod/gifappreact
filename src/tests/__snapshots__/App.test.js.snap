// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing in App.js Component <App/> should show correctly 1`] = `
<div
  className="container"
>
  <h1>
    Gif App React
  </h1>
  <SearchInput
    handleQuery={[Function]}
  />
  <div
    className="content"
  >
    <GifGrid
      query=""
    />
  </div>
</div>
`;
